src/components/multi-action-button/multi-action-button.test.js

```diff
@@ -0,0 +1,199 @@
+      (alignment) => {
+        CypressMountWithProviders(<MultiActionButtonList align={alignment} />);
+
+        multiActionButtonComponent().should("have.css", `margin-${alignment}`);
+      }
+    );
+
+    it("should check Multi Action Button is disabled", () => {
+      CypressMountWithProviders(<MultiActionButtonList disabled />);
+
+      multiActionButtonComponent().trigger("mouseover");
+      multiActionButtonList()
+        .eq(0)
+        .should("be.disabled")
+        .and("have.attr", "disabled");
+      multiActionButtonList()
+        .eq(1)
+        .should("be.disabled")
+        .and("have.attr", "disabled");
+      multiActionButtonList()
+        .eq(2)
+        .should("be.disabled")
+        .and("have.attr", "disabled");
+    });
+
+    it("should invoke Multi Action Button component and expands and contains three items", () => {
+      CypressMountWithProviders(<MultiActionButtonList />);
+
+      multiActionButtonComponent().trigger("mouseover");
+      multiActionButtonList()
+        .eq(0)
+        .should("have.text", "Example Button")
+        .and("be.visible");
+      multiActionButtonList()
+        .eq(1)
+        .should("have.text", "Example Button with long text")
+        .and("be.visible");
+      multiActionButtonList()
+        .eq(2)
+        .should("have.text", "Short")
+        .and("be.visible");
+    });
+
+    it("should render Multi Action Button with golden border", () => {
+      CypressMountWithProviders(<MultiActionButtonList />);
+
+      pressTABKey(1);
+      multiActionButtonComponent()
+        .children()
+        .should("have.css", "border", "3px solid rgb(255, 181, 0)");
+    });
+
+    it("should render Multi Action Button with golden border", () => {
+      CypressMountWithProviders(<MultiActionButtonList />);
+
+      multiActionButtonComponent().trigger("mouseover");
+      multiActionButton().should(
+        "have.css",
+        "background-color",
+        "rgb(0, 77, 42)"
+      );
+    });
+
+    it("should call click event", () => {
+      const callback = cy.stub();
+
+      CypressMountWithProviders(<Button onClick={callback} />);
+
+      buttonDataComponent()
+        .click({ force: true })
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          expect(callback).to.have.been.calledOnce;
+        });
+    });
+
+    it("should invoke Multi Action Button component in a hidden container", () => {
+      CypressMountWithProviders(
+        <Accordion title="Heading">
+          <MultiActionButton
+            size="large"
+            subtext="subtext"
+            text="Multi Action Button"
+          >
+            <Button size="large">Button 1</Button>
+            <Button size="large">Button 2</Button>
+            <Button size="large">Button 3</Button>
+          </MultiActionButton>
+        </Accordion>
+      );
+
+      accordionDefaultTitle()
+        .trigger("keydown", { force: true }, keyCode("Enter"))
```

Comment:
could we not use `{force:true}`? after run the test the `MultiActionButton` is not visible, because the `Accordion` is not expanded